Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs

```diff
@@ -227,18 +228,14 @@ public IEnumerable<UserAc> GetAllUsers()
         /// <returns>List of all Employees</returns>
         public async Task<List<UserAc>> GetAllEmployees()
         {
-            var employees = await _userManager.GetUsersInRoleAsync(_stringConstant.RoleEmployee);
+            var users =await _userManager.Users.Where(user=>user.IsActive==true).ToListAsync() ;
```

Comment:
we never compare boolean. IsActive itself will return true or false. There is no need for comparison.